Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp

```diff
@@ -1637,3 +1689,38 @@ GList* webkit_website_data_manager_get_itp_summary_finish(WebKitWebsiteDataManag
 
     return static_cast<GList*>(g_task_propagate_pointer(G_TASK(result), error));
 }
+
+/**
+ * webkit_website_data_manager_clear_origin_storage_ratio:
+ * @manager: a #WebKitWebsiteDataManager
+ *
+ * Reset existing origin storage ratio.
+ *
+ * Resetting origin storage ratio causes that quota is based on perOriginStorageQuota.
```

Comment:
This is confusing, what is `perOriginStorageQuota`? I don't think we want functions to reset the value, it's a construct only property, it shouldn't change. I think resetting is probably needed for WKTR to ensure changing this in one tests don't affect others, but I don't think we need this in the public API.

Reply:
You are right, I will remove these functions for clearing.